docs/LearningOutcomes.adoc

```diff
@@ -0,0 +1,216 @@
+= Learning Outcomes
```

Comment:
This part can delete?

Reply:
Yes I have deleted it already, thank you for the reminder!